### ISSUE TITLE
Playbook Page Hierarchy

### ISSUE THREAD

**Issue by steveharrison** [qantasairways/playbook#8]
I'm working to improve the styling of the Playbook, and something I think that would be good is if we don't have any page content associated with the top-level items and have them just for grouping. This will simplify the navigation menu and we can just have a simple list of items with chevrons containing page items.

An example of this structure: https://developer.apple.com/design/human-interface-guidelines/macos/user-interaction/authentication/.

**Comment by paulkeen** [comment]
That's fine but would need a little change to the MD files. I'd be happy to do when you've finished the styling. Thanks for helping out.

**Comment by aMollusk** [comment]
Yeah, it seems that the existing content in the top level pages isn't all that valuable. I'm on board with this.

**Comment by paulkeen** [comment]
Appreciate your help with all this, How's it tracking along?
